docs/src/assets/scss/components/alert.scss

```diff
@@ -63,27 +45,15 @@
     margin-block-end: .25rem;
 
     .alert--warning & {
-        color: var(--color-rose-700);
-
-        [data-theme="dark"] & {
-            color: var(--color-rose-200);
-        }
+        color: var(--alert-warning-heading-color);
     }
 
     .alert--important & {
-        color: var(--color-warning-700);
-
-        [data-theme="dark"] & {
-            color: var(--color-warning-200);
-        }
+        color: var(--alert-important-heading-color);
     }
 
     .alert--tip & {
-        color: var(--color-success-700);
```

Comment:
The text `Tip` was slightly bolder earlier.

### after
<img width="80" alt="Screenshot 2024-09-19 at 3 49 31 PM" src="https://github.com/user-attachments/assets/fa0399d0-21ee-4386-89be-792357bcec1c">

### before

<img width="70" alt="Screenshot 2024-09-19 at 3 49 54 PM" src="https://github.com/user-attachments/assets/434c5f68-98d0-4bd4-aebb-e5751c35d109">

Reply:
https://github.com/user-attachments/assets/d90f4a26-b27f-4c0f-8ce4-a63c9c6f312d

Reply:
![image](https://github.com/user-attachments/assets/c262be8f-b9f8-4a11-81ab-b6fd2c5bdb3f)

Left image- js disabled - https://deploy-preview-18916--docs-eslint.netlify.app/use/configure/configuration-files
Right image- js enabled - https://eslint.org/docs/latest/use/configure/configuration-files
I don't see any difference in CSS 🤔

Reply:
Could it be some global style interference?

Reply:
figuring out this, right now `font-weight: 500` seems not be working in no-js mode, but `font-weight: 600` is.

Reply:
Could the change in `font-weight` be due to some fonts not loading properly in no-JS mode?

Reply:
Just a gentle reminder: this is the only item pending. When you’re back, please take a look. Thank you!

Reply:
I'm not exactly sure why the font-weight appears slightly lighter in no-JS mode.

Reply:
https://github.com/eslint/eslint/blob/f16e846ac004bc32e52cd3991d14d7a89374bbb5/docs/src/_includes/layouts/base.html#L108-L115

What i can see here is we are using font with the `regular` font-weight which is not supporting `font-weight:500` in no-js mode, so as a solution we can use the variable font like `Inter-VariableFont_slnt\,wght.ttf` which is working properly.

But i would like to here some opinions on this solutions, may be the current fonts are serving some purpose i am not aware of.

please share if solution seems right.

Reply:
I tried to fix the font-weight issue of alert's heading can you please check once if this way seems correct? So we can implement the fix with other components also as there are many texts with this issue like in buttons:
![Screenshot 2024-10-25 223253](https://github.com/user-attachments/assets/241110de-68c0-4069-b7a5-0466dad15db0)

Reply:
Sure. I will check.